WifiART/src/structs/ImageTypes.java

```diff
@@ -54,8 +58,22 @@
     private ImageTypes() {}
 
     /**
-     * Returns a vdriable-length list containing the image types.
-     * @return
+     * Returns a HashMap mapping the image types to their index/encoding.
+     *
+     * @return the map mapping image types to their encoding.
+     */
+    private static HashMap<String, Byte> getImageTypesMap() {
+        HashMap<String, Byte> imageTypesMap = new HashMap<>();
+        for (int i = 0; i < IMAGE_TYPES.length; i++) {
+            imageTypesMap.put(IMAGE_TYPES[i], (byte) i);
```

Comment:
Does this mean the order of the constants in the IMAGE_TYPES matter? Would it make sense to use enum for IMAGE_TYPES in this case?

Reply:
I decided not to use an enum in this case because this would require very long switch-statements to dictate behavior based on the value that the user chooses. In this case the order does matter, because I am using the index to represent the integer value of the encoding. I'm sure there are other ways of implementing this, but I chose this way to minimize the amount of code logic needed.

Reply:
I see. Thanks.